### DISCUSSION TITLE
Game selection

### DISCUSSION THREAD

**Discussion by PepperBacon** [ebkr/r2modmanPlus#689] [Q&A]
R2modman it keeps launching as the game muck but i want mods for Risk of rain 2 how to I change what game I want modded Restarting/ reinstalling R2modman does nothing.

**Comment by ebkr** [comment]
Settings -> Change game

**Comment by thettenhausen** [comment]
Resurrecting this, sorry... Just came here after seeing that restarting the app for switching games was no longer necessary, and I was really wondering if I was stupid, because I could not find a way to switch games or profiles.
Could this be moved as menu entries to the left? Seems like a convoluted way and semantically not related to the setting of the app, the current game, or the current profile.

**Reply by MythicManiac** [reply]
Currently there's a fair bit of refactoring work ongoing with the aim of enabling the navigation menu to be more easily modified, since we have plans to rework it in a major way for the Thunderstore Mod Manager. Whether or not similar changes are incorporated to r2modman is ultimately up to Ebkr, but at the very least it should be somewhat easier soon.
